resources/scponly_user.rb

```diff
@@ -90,8 +90,10 @@
     end
   end
 
-  group 'scponly' do
-    members new_resource.name
+  temp = new_resource.name
+
+  edit_resource(:group, 'scponly') do
+    members temp
   end
```

Comment:
So here's how we should fix this:
```ruby
  group "scponly append #{new_resource.name}" do
    group_name 'scponly'
    append true
    members new_resource.name
    action :modify
  end
```

Then in the default recipe, change ``group[scponly]`` to the following (adding the append parameter is key:
```ruby
group 'scponly' do
  append true
  system true
end
```